### ISSUE TITLE
Make the fork on github and download links more prominent

### ISSUE THREAD

**Issue by dpsnowden** [ioos/notebooks_demos#213]
A suggestion from Roy, (not sure gh-handle).

Scrolling all the way to the bottom for a text link to the actual notebook buries the lead.  What can we do to make the download link more evident?

**Comment by ocefpaf** [comment]
I have to be honest that I kind of like it at the bottom of the page. IMO it avoids distractions until the person had the time to read the page (or at least scroll all the way to the end).

Clicking to open/download will lead to the same "text", only without the page theme. The only "extra" is to be able to save it locally. So I don't see the need to put that in evidence.

In the example https://rmendels.github.io/pices2017.nb.html there is a JS button to show/hide code\* and a download item. Making something similar would mean a lot of code on the `nbconverter` side (but I will check with nbconvert guys if they are interested).

The "quick" solution would be to move the bottom cell to the top.

\* we don't really want that part because the code is part of the narrative in our case

**Comment by jbosch-noaa** [comment]
So this issue was opened a month ago but I forgot to mention to you, @ocefpaf , that I like with Roy Mendelson did to his notebook page.   It may be a bit of code but I would like us to work towards it.  I agree with @dpsnowden that we should make the downloading of the notebook more prominent.

**Comment by ocefpaf** [comment]
I looked into the possible options at the time and I could not fine a nice solution that wouldn't require massive changes into the theme. I will investigate again if we can do this using `nbconvert` and a clever template instead.

**Comment by jbosch-noaa** [comment]
@ocefpaf  - I talked with @mwengren  and he suggested that we just put the "download notebook" link at the top of the page using a modification to the front page layout.   So Micah suggests that you go here: 

ioos_jekyll_theme/_layouts/notebook.html 

and add an additional div tag with the HTML for the notebook download link  in between lines 23 and 24:

ln 23 **</script>**
_Add div code here_ 
ln 24 **<  div id="content">**

He then said you would need to make a pull request to make the change but that's ok.  Does this make sense?

**Comment by ocefpaf** [comment]
Not sure. I'll test that. I wanted to avoid theme changes and I am looking if we can accomplish the same on the converter side.

**Comment by ocefpaf** [comment]
Note to self: check if https://github.com/jupyter/nbviewer/pull/681 can help to do this using only `nbconvert` and minimal changes to the `theme`.

**Comment by dpsnowden** [comment]
@ocefpaf this was a "nice to have" suggestion from a colleague.  If the implementation would require significant changes to nbconvert then I suggest that it's out of scope for now.  Let's concentrate on the actual notebooks now and polish the foundational tools later.  In general I'm in favor of more notebooks over perfecting any one notebook.  

That said,  if you have a way to file this issue appropriately so that at some point in the future we can decide if it's worth it to focus a release entirely on presentation and not content, I'm fine with that type of planning.  Better yet get the `nbconvert` developers to incorporate it into their planning. 

Last, I don't understand why this is labeled `ready` if there hasn't been any development yet.  If you agree with my logic, can you move this back to the backlog and label appropriately?

**Comment by ocefpaf** [comment]
>  In general I'm in favor of more notebooks over perfecting any one notebook.

Sure. The new notebooks are stalled at the moment b/c I am at SciPy this week. I do have some that are 95% ready. PRs next week...

> Last, I don't understand why this is labeled ready if there hasn't been any development yet. If you agree with my logic, can you move this back to the backlog and label appropriately?

I do agree. I guess that is waffle automatic labeling. Maybe @jbosch-noaa knows more about it.

**Comment by jbosch-noaa** [comment]
The "ready" column is for projects that are ready to be worked on.  I moved this one to the backlog column for now.  Maybe I can rename these columns to better match our Trello board ....

**Comment by MathewBiddle** [comment]
@ocefpaf, is this something to explore again as we are now in the 'future'?

**Comment by ocefpaf** [comment]
> @ocefpaf, is this something to explore again as we are now in the 'future'?

We add the fork banner that is "kind of" the standard for sites on GH pages. If that is enough I don't guess we need one for every notebook. The original suggestion was a comparison to what RMarkdown does and, IMO, that does not really transfer well to a Jupyter collection of notebooks.

**Comment by MathewBiddle** [comment]
In looking at [other resources](https://plotly.com/python/pca-visualization/), it would be nice to have the GH link (or link to the raw .ipynb) and binder link at the top:
![image](https://user-images.githubusercontent.com/8480023/101668467-95b48e00-3a1e-11eb-9e4b-6a5cb73dcfbe.png)

TBH, before starting at IOOS, I would go to the notebook of interest and want to either 1) download the .ipynb and run it locally, or 2) go to the binder instance. 

The fork banner is attempting to get contribution, which I don't think this is necessarily after.

**Comment by MathewBiddle** [comment]
I think JupyterBook handles this nicely. 

rocket - points to binder and colab
[ ] - makes it full screen
github - open a ticket
download - download ipynb or pdf of the page

![image](https://user-images.githubusercontent.com/8480023/125313321-1fa13e00-e303-11eb-93cf-b09ad7ff7d9d.png)

closed with https://github.com/ioos/notebooks_demos/pull/409
